tests/bluetooth/classic/sdp_c/testcase.yaml

```diff
@@ -0,0 +1,16 @@
+tests:
+  sdp.c:
+    platform_allow:
+      - native_sim
+      - native_sim/native/64
+    integration_platforms:
+      - native_sim
+    tags:
+      - bluetooth
+      - sdp
+    harness: pytest
+    harness_config:
+      pytest_dut_scope: session
+      fixture: bumble
+      pytest_args: ['--log-level=DEBUG']
```

Comment:
I believe setting log level to debug shouldn't be a default for a test. If you are interested in debugging you should do this locally, passing the pytest_arg in twister's CLI

Reply:
OK. I will remove it.

Reply:
updated